src/databaseInterface/src/main/kotlin/org/vaccineimpact/api/db/CloseableContext.kt

```diff
@@ -0,0 +1,33 @@
+package org.vaccineimpact.api.db
```

Comment:
I don't think my naming is very good here. Any suggestions?

Reply:
This seems ok, I can't think of anything better really!